docs/reference/modules/attachment.md

```diff
@@ -10,75 +10,120 @@ extends: '@apostrophecms/module'
 
 The `attachment` module coordinates the addition and manipulation of any files added to the database and works alongside the `@apostrophecms/uploadfs` module. This includes rescaling and cropping of images, as well as the uploading of non-image files like '.pdf' or '.csv'. It exposes multiple template helpers and command line tasks for attachment retrieval and manipulation.
 
-Options for this module are passed in through both `modules/@apostrophecms/attachment/index.js` and `modules/@apostrophecms/uploadfs/index.js`.
+Options and configuration settings for this module are passed in through both `modules/@apostrophecms/attachment/index.js` and `modules/@apostrophecms/uploadfs/index.js`.
 
 ## Options
 
-The `@apostrophecms/attachment` module is unusual compared to other modules in that some settings are passed directly to `@apostrophecms/uploadfs`. Those options are marked with a '*' in the options table. These options should be added to the `modules/@apostrophecms/uploadfs/index.js` file inside the `uploadfs` object. Additionally, the `imageSizes` property is passed as a top-level property of the `/modules/@apostrophecms/attachment/index.js` file, not in the `options` object.
-
+The `@apostrophecms/attachment` module is unusual compared to other modules in that some settings are passed directly to `@apostrophecms/uploadfs`. For these, see the [selected settings](#selecteduploadfssettings) of the `@apostrophecms/uploadfs` module section. 
```

Comment:
How about "this reference is unusual compared to others..." It's not the module that's unusual.

Reply:
Changed and hopefully clarified.